.changeset/fifty-sloths-march.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux-private/preview-middleware-client': patch
+'@sap-ux/preview-middleware': patch
+---
+
+call syncOutline in the initOutline because the attached event is not triggered for every case(if the app is already loaded before the event is attac
```

Comment:
Please change to "fix outline initialisation for the case when application is loaded, but outline is empty"

Reply:
Done